ffi/src/lib.rs

```diff
@@ -657,7 +1386,7 @@ pub unsafe extern "C" fn typed_value_list_into_iter(values: *mut Vec<Binding>) -
 /// A destructor `typed_value_destroy` is provided for releasing the memory for this
 /// pointer type.
 #[no_mangle]
-pub unsafe extern "C" fn typed_value_list_iter_next(iter: *mut BindingIterator) -> *mut Binding {
+pub unsafe extern "C" fn typed_value_list_iter_next(iter: *mut BindingIterator) ->  *mut Binding {
```

Comment:
I personally found this style (two spaces after `->`) odd, but now I see you're doing it as a change.  Why do you want the two spaces?

Reply:
No, this was a rebase issue. I fixed it in the parent branch and I think the rebase of this branch onto this caused it and I missed it when I reviewed.